design/diagrams/kots_install_sequence.mmd

```diff
@@ -0,0 +1,267 @@
+sequenceDiagram
+	participant kots as Kots CLI
+	participant k8s as Cluster k8s API
+	participant kotsadm as Kots Admin API
+	participant kotsdb as Kots Admin Database (rqlite)
+	participant kotsstore as Kots Admin Data Store (minio)
+	participant docker as Docker Registry
+	participant repli as Replicated API
+
+	%% Check that there are sufficient RBAC permissions on the cluster
+	note over kots, k8s: Check that there are sufficient priveledges on the cluster to deploy the app
+	kots -> k8s: Self Subject Access Review of resource attributes [Verb: *, Group: *, Version: *, Resource: *]
```

Comment:
`->` doesn't render an arrowhead; `->>` and `-->>` [will](https://mermaid.js.org/syntax/sequenceDiagram.html#messages) though.